pkg/webhook/pod/webhook.go

```diff
@@ -102,7 +103,7 @@ func Mutate(pod *corev1.Pod, sliceName string) *corev1.Pod {
 
 	// Add vl3 annotation to pod template
 	annotations := pod.ObjectMeta.Annotations
-	annotations[nsmInjectAnnotaionKey] = "vl3-service-" + sliceName
+	annotations[nsmInjectAnnotaionKey] = fmt.Sprintf("kernel://vl3-service-%s/nsm0", sliceName)
```

Comment:
does this work only with kernel forwarder? or will it work with vpp as well?

Reply:
I guess for now atleast our goal is the kernel forwarder. When we support other forwarders, we can look if we need to change this.